### ISSUE TITLE
[Feature Request]  Search project's wiki

### ISSUE THREAD

**Issue by ThomasAkam** [CenterForOpenScience/osf.io#9210]
There does not appear to be any way to search within the wiki of a project.  This makes it hard to find relevent information once the wiki has grown beyond a small size.  Please consider adding functionallity to search a project's wiki.  Ideally, search within a project would return relevent pages from the wiki and also files with matching filenames.

**Comment by saradbowman** [comment]
Hi @ThomasAkam thanks for getting in touch. You're right, there is not currently a way to do this. We are always gathering feedback from users about what would make OSF more useful to them, and I'll log your feature request in our internal tracker. Don't hesitate to reach out with more feedback.

**Comment by yves-weissenberger** [comment]
yes! I think this would be a hugely valuable addition. Once you have a sizeable wiki that new lab members could be referred to for most issues it becomes very difficult (for those new lab members) to find relevant information.

**Comment by MariosPanayi** [comment]
> Hi @ThomasAkam thanks for getting in touch. You're right, there is not currently a way to do this. We are always gathering feedback from users about what would make OSF more useful to them, and I'll log your feature request in our internal tracker. Don't hesitate to reach out with more feedback.

A search feature would be an immensely helpful quality of life improvement for the wiki! Struggling without one at the moment.

**Comment by KristinaRiemer** [comment]
I'm another OSF user who thinks search functionality for wikis would be great and useful!

**Comment by ThomasAkam** [comment]
Hi @saradbowman & @brianjgeiger, is there any plan to implement a search for the wiki?  It seems like there is quite a bit of interest in this feature.  I was just reminded of this issue when I had to direct a new lab member to information on our lab wiki as they could not find it themselves..  Thanks!
